src/NServiceBus.Core/Unicast/ContextualBus.cs

```diff
@@ -254,27 +268,28 @@ List<string> GetAtLeastOneAddressForMessageType(Type messageType)
             return addresses;
         }
 
-      
-        void SendMessage(Type messageType, object message, ExtendableOptions options)
+
+        void SendMessage(Type messageType, object message, NServiceBus.SendOptions options)
```

Comment:
Yes there is another one https://github.com/Particular/NServiceBus/blob/develop/src/NServiceBus.Core/v6-obsoletes.cs#L198